src/oca_pre_commit_hooks/checks_odoo_module.py

```diff
@@ -169,7 +169,8 @@ def lookup_manifest_paths(filenames_or_modules):
     return odoo_module_files_changed
 
 
-def run(files_or_modules, enable=None, disable=None, no_verbose=False, no_exit=False, list_msgs=False):
+def run(files_or_modules, enable=None, disable=None, no_verbose=False, no_exit=False, list_msgs=False, autofix=False):
+    _ = autofix
```

Comment:
This is temporal, isn't it?

Reply:
Yes, it was a mistake, I now use it as an argument for the constructor.